packages/utils/src/crypto_utils.go

```diff
@@ -0,0 +1,230 @@
+package utils
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"log"
+
+	"strconv"
+
+	"github.com/kartikaysaxena/substrateinterface/signature"
```

Comment:
Please change these in all PRs.

Reply:
actually, I am testing this through the `kartikaysaxena/substrateinterface` which is just a copy of the code of the draft PR in the library, would change this once the PR in the client gets merged, good for testing purposes, still if I would replace this with the original client if you ask me to do it.

Reply:
Sure.